docs/scripting/resources/weaponids.md

```diff
@@ -0,0 +1,71 @@
+| ![](/images/hudIcons/countryRifle.png)<br/>Country Rifle                      | ![](/images/deathIcons/death-countryRifle.gif)        | WEAPON_RIFLE              | 33        | 6         | 357           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/sniperRifle.png)<br/>Sniper Rifle                        | ![](/images/deathIcons/death-sniperRifle.gif)         | WEAPON_SNIPER             | 34        | 6         | 358           | Skill can be set with [SetPlayerSkillLevel](../functions/SetplayerSkillLevel)                                         |
+| ![](/images/hudIcons/rpg.png)<br/>RPG                                         | ![](/images/deathIcons/death-rpg.gif)                 | WEAPON_ROCKETLAUNCHER     | 35        | 7         | 359           | -                                                                                                                     |
+| ![](/images/hudIcons/hsRocket.png)<br/>HS Rocket                              | ![](/images/deathIcons/death-hsRocket.gif)            | WEAPON_HEATSEEKER         | 36        | 7         | 360           | Lock-on is not synced                                                                                                 |
+| ![](/images/hudIcons/flameThrower.png)<br/>Flamethrower                       | ![](/images/deathIcons/death-flameThrower.gif)        | WEAPON_FLAMETHROWER       | 37        | 7         | 361           | -                                                                                                                     |
+| ![](/images/hudIcons/minigun.png)<br/>Minigun                                 | ![](/images/deathIcons/death-minigun.gif)             | WEAPON_MINIGUN            | 38        | 7         | 362           | -                                                                                                                     |
+| ![](/images/hudIcons/satchelCharge.png)<br/>Satchel Charge                    | ![](/images/deathIcons/death-satchelCharge.gif)       | WEAPON_SATCHEL            | 39        | 8         | 363           | Only synced for players that were streamed-in when the satchels were thrown                                           |
+| ![](/images/hudIcons/detonator.png)<br/>Detonator                             | ![](/images/deathIcons/death-detonator.gif)           | WEAPON_BOMB               | 40        | 12        | 364           | Given automatically when players throw a satchel charge (omit from anti-cheat checks)                                 |
+| ![](/images/hudIcons/sprayCan.png)<br/>Spraycan                               | ![](/images/deathIcons/death-sprayCan.gif)            | WEAPON_SPRAYCAN           | 41        | 9         | 365           | Players that are sprayed choke                                                                                        |
```

Comment:
`/images/hudIcons/sprayCan.png` fails to load

Reply:
that's interesting because everything loads for me